xml/http_servers.xml

```diff
@@ -2390,12 +2390,11 @@
        variety of products including printers, PDUs, etc.
   -->
 
-  <fingerprint pattern="^\$ProjectRevision: 4.0.2.38 \$$">
-    <description>This banner is seen on some HP LaserJet printers.</description>
+  <fingerprint pattern="^\$ProjectRevision: \d.\d.\d.\d\d \$$">
+    <description>This banner is used to fingerprint devices for Ripple20</description>
     <example>$ProjectRevision: 4.0.2.38 $</example>
```

Comment:
If you end up capturing the version please make sure the example tests the capture.

As a note, here are the version distributions we saw from a Project Sonar scan:

```
 625 "$ProjectRevision: 6.0.1.5 $"
 235 "$ProjectRevision: 6.0.1.45 $"
 102 "$ProjectRevision: 4.0.2.38 $"
  41 "$ProjectRevision: 4.7.1.12 $"
  34 "$ProjectRevision: 5.0.1.23 $"
  31 "$ProjectRevision: 6.0.1.29 $"
  23 "$ProjectRevision: 4.7.1.17 $"
  23 "$ProjectRevision: 4.2 $"
  16 "$ProjectRevision: 4.2.2.12 $"
  11 "$ProjectRevision: 4.1.2.11 $"
   6 "$ProjectRevision: 5.0.1.5 $"
   4 "$ProjectRevision: 6.0.1.46 $"
   4 "$ProjectRevision: 4.5.1.10 $"
   2 "$ProjectRevision: 6.0.1.60 $"
   1 "$ProjectRevision: 5.0.1.31 $"
   1 "$ProjectRevision: 4.7.1.26 $"
   1 "$ProjectRevision: 4.7.1.24 $"
   1 "$ProjectRevision: 4.2.2.5 $"
```